app/views/DependencyExplorerPage.scala.html

```diff
@@ -59,7 +59,7 @@ <h1 id="search-service-header">Dependency Explorer</h1>
                 <div class="row" style="margin-bottom: 20px;">
                     <div class="col-md-4">
                         <label for="group-artefact-search">Group:Artefact</label>
-                        <input id="group-artefact-search" class="form-control" type="text" value="@DependencyExplorerController.groupArtefactFromForm(form)" onfocus="clearSearch()" autocomplete="off">
+                        <input id="group-artefact-search" class="form-control" type="text" value="@DependencyExplorerController.groupArtefactFromForm(form)" autocomplete="off">
```

Comment:
BDOG-2354 Removing the clear on focus behaviour from Dependency Explorer group/artefact